Packs/PhishLabs/Scripts/PhishLabsPopulateIndicators/PhishLabsPopulateIndicators.py

```diff
@@ -78,12 +74,12 @@ def main():
                 if indicator.get('createdAt'):
                     indicator_timestamp = datetime.strptime(indicator['createdAt'], '%Y-%m-%dT%H:%M:%SZ')
                 if indicator_type == 'Attachment':
-                    indicator_type = 'File MD5'
+                    indicator_type = FeedIndicatorType.File
```

Comment:
I think it's better to get the indicator type from `indicator_type_finder` and remove this assignment.

Reply:
ok, then two questions - are there any other "PhishLabs indicator types" that can be classified to File other than "Attachment"? and are they all md5?